lagom-eventstore-example/lagom-eventstore-impl/src/main/resources/log4j.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/" debug="true">
```

Comment:
Does this mean that both logback and log4j are in use? Would it be better to use [log4j-over-slf4j](https://www.slf4j.org/legacy.html#log4j-over-slf4j)?

Reply:
This is still confusing me a bit. All the dependences pulled in via IBM's EventStore seem to be log4j friendly. This file is a leftover from a previous attempt I did at configuring the output.

I will give it a couple more attempts but probably just remove this file and Lagom's logback dependency and just go with that.

Reply:
I've left a `TODO` note in the `pom.xml` and probably remove all logging in a future iteration